src/scanner.py

```diff
@@ -2,6 +2,7 @@
 from token_type import TokenType
 
 
+
```

Comment:
Avoid adding extra space and removing space, you can use `autopep8` or similar extension in the code editor.